core/eolearn/tests/test_core_tasks.py

```diff
@@ -172,47 +173,45 @@ def test_add_rename_remove_feature(patch):
     assert len(patch.mask) == 0, "mask features were not removed"
 
 
-def test_duplicate_feature(patch):
-    mask_data = np.arange(10).reshape(5, 2, 1, 1)
-    feature_name = "MASK1"
-    duplicate_name = "MASK2"
+@pytest.mark.parametrize(
+    "feature_specification",
+    [
+        [(FeatureType.DATA, "bands", "bands2")],
+        [(FeatureType.DATA, "bands", "bands2"), (FeatureType.MASK_TIMELESS, "mask", "mask2")],
+        [(FeatureType.DATA, "bands", f"bands{i}") for i in range(5)],
+    ],
+)
+def test_duplicate_feature(feature_specification: List[FeaturesSpecification], patch: EOPatch) -> None:
+    patch = DuplicateFeatureTask(feature_specification)(patch)
 
-    patch = AddFeatureTask((FeatureType.MASK, feature_name))(patch, mask_data)
+    for f_type, f_name, f_dup_name in feature_specification:
+        assert f_dup_name in patch[f_type]
+        assert id(patch[(f_type, f_name)]) == id(patch[(f_type, f_dup_name)])
```

Comment:
add assertion message why this must hold

Reply:
done